packages/core/src/io/writer-context.ts

```diff
@@ -111,9 +111,19 @@ export class WriterContext {
 		accessorDef.type = accessor.getType();
 		accessorDef.componentType = accessor.getComponentType();
 		accessorDef.count = accessor.getCount();
-		accessor.getMax((accessorDef.max = []));
-		accessor.getMin((accessorDef.min = []));
-		accessorDef.normalized = accessor.getNormalized();
+
+		const needsBounds = this._doc.getGraph()
+			.listParentLinks(accessor)
+			.some((link) => link.getName() === 'POSITION');
+		if (needsBounds) {
+			accessor.getMax((accessorDef.max = []).map(Math.fround));
+			accessor.getMin((accessorDef.min = []).map(Math.fround));
```

Comment:
If `accessor.getMax` fetches values from binary buffers, then `Math.fround` is a no-op.

Reply:
Yes, missed this! Fixed in https://github.com/donmccurdy/glTF-Transform/commit/dedd34c1b78986a214b4730149f3c29799146bd4.